dist/jquery.popup-interactive-world-map.coffee

```diff
@@ -0,0 +1,49 @@
+$ ($, window, document) ->
+  pluginName = 'interactiveWorldMapCountryPopup'
+  settings   = null
+  element    = null
+  mainModal  = null
+
+  defaults =
+    popup:
+      margins:
+        top:  100
+        left: 60
+
+  Popup = (options) ->
+    @_options = options;
+    @init();
+
+  $.extend Popup.prototype,
+    init: ->
+      settings  = $.extend defaults, @_options
```

Comment:
Personal opinion: use `()` for function calls like this. I find it more readable :tongue: